src/tests/system/tests/test_sssctl.py

```diff
@@ -286,44 +177,46 @@ def test_sssctl__check_invalid_option_name_in_snippet(client: Client):
     :customerscenario: False
     """
     client.sssd.common.local()
-    client.fs.write("/etc/sssd/conf.d/01_snippet.conf", "[domain/local]\ninvalid_option = True", mode="600")
+    client.fs.write("/etc/sssd/conf.d/01_snippet.conf", contents, mode="600")
```

Comment:
For sssd-2.10+ it is 'rw-r-----', i.e. 640

Won't be visible though, since sssd.service chmod-s everything...